randovania/generator/hint_distributor.py

```diff
@@ -59,18 +96,11 @@ def get_generic_hint_nodes(self, prefill: PreFillParams) -> list[NodeIdentifier]
             if isinstance(node, HintNode) and node.kind == HintNodeKind.GENERIC
         ]
 
-    async def get_specific_pickup_precision_pair_overrides(
-        self, patches: GamePatches, prefill: PreFillParams
-    ) -> dict[NodeIdentifier, PrecisionPair]:
+    async def get_specific_pickup_precision_pairs(self) -> dict[NodeIdentifier, PrecisionPair]:
```

Comment:
docstring. Should mention it needs a key for every HintNode with kind `SPECIFIC_PICKUP`